src/navigation/navigation_main.cc

```diff
@@ -62,30 +63,29 @@ using std::string;
 using std::vector;
 using Eigen::Vector2f;
 
-DEFINE_string(laser_topic, "velodyne_2dscan", "Name of ROS topic for LIDAR data");
-DEFINE_string(odom_topic, "jackal_velocity_controller/odom", "Name of ROS topic for odometry data");
-DEFINE_string(loc_topic, "localization", "Name of ROS topic for localization");
-DEFINE_string(init_topic,
-              "initialpose",
-              "Name of ROS topic for initialization");
-// Name of topic controlling whether to enable autonomous nav or not.
-DEFINE_string(enable_topic, "autonomy_arbiter/enabled",
-    "ROS topic that indicates whether autonomy is enabled or not.");
-DEFINE_string(map,
-              "maps/Joydeepb-Home/Joydeepb-Home.navigation.txt",
-              "Name of navigation map file");
-DECLARE_string(helpon);
+const string kAmrlMapsDir = ros::package::getPath("amrl_maps");
```

Comment:
An existence check could be nice here, informing the user if `amrl_maps` cannot be found.